libs/community/langchain_community/chat_models/dappier.py

```diff
@@ -66,21 +66,30 @@ class ChatDappierAI(BaseChatModel):
 
     dappier_endpoint: str = "https://api.dappier.com/app/datamodelconversation"
 
-    dappier_model: str = "dm_01hpsxyfm2fwdt2zet9cg6fdxt"
+    dappier_model: str = Field("dm_01hpsxyfm2fwdt2zet9cg6fdxt", alias="model")
 
-    dappier_api_key: Optional[SecretStr] = Field(None, description="Dappier API Token")
+    dappier_api_key: Optional[SecretStr] = Field(
+        None, description="Dappier API Token", alias="api_key"
+    )
 
     class Config:
         """Configuration for this pydantic object."""
 
         extra = Extra.forbid
+        allow_population_by_field_name = True
+        arbitrary_types_allowed = True
 
-    @root_validator()
+    @root_validator
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that api key exists in environment."""
-        values["dappier_api_key"] = convert_to_secret_str(
-            get_from_dict_or_env(values, "dappier_api_key", "DAPPIER_API_KEY")
-        )
+        api_key = values["dappier_api_key"] or values["api_key"]
```

Comment:
don't need to check "api_key" (it will have been moved to dappier_api_key by this pt)